src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -84,6 +84,9 @@ export interface IAnalyticsFacetMeta {
   facetId: string;
   facetField: string;
   facetValue?: string;
+  facetRangeStart?: string;
+  facetRangeEnd?: string;
+  facetRangeEndInclusive?: boolean;
```

Comment:
Following from the planif, is this boolean parameter being used by the UA/ML teams? If not, could we remove it?

Reply:
They are not using them now but they can be useful in the future